tests/test_endf.py

```diff
@@ -1167,7 +1167,7 @@ def test_evaluation_electroatomic():
     assert_array_almost_equal(brem.xs(E), xs)
 
     # Check bremsstrahlung secondary distributions
-    assert brem.products[0]["za"] == 11  # electrons
```

Comment:
This should be an integer comparison and should be exact.  Maybe the issue is that elsewhere in PyNE we need to make sure this is converted from float to integer, since it's probably formatted as a float in ENDF even though it's an integer.